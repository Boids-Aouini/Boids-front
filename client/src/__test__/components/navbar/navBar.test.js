import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store';
import NavBar from '../../../components/navbar/navBar';

let mockStore = configureStore();
let initState = {};
let store = mockStore(initState);

describe('<NavBar />', () => {
    describe('render()', () => {
        test('should render without crashing', () => {
            shallow(<NavBar store={store} />)
        })
        test('should render component', () => {
            let wrapper = shallow(<NavBar store={store} />)
            let component = wrapper.dive();
            expect(toJson(component)).toMatchSnapshot();
        })


    })

})