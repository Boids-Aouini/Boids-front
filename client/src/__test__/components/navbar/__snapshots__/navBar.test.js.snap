// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavBar /> render() should render component 1`] = `
<NavBar
  dispatch={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
